interactive_engine/groot-module/src/main/java/com/alibaba/graphscope/groot/store/StoreService.java

```diff
@@ -298,6 +309,23 @@ private Map<Integer, OperationBatch> writeStore(
         return batchNeedRetry;
     }
 
+    private JsonObject buildMetricJsonLog(
+            boolean succeed, OperationBatch operationBatch, long start, int partitionId) {
+        JsonObject metricJsonLog = new JsonObject();
+        String traceId = operationBatch.getTraceId();
+        long current = System.currentTimeMillis();
+        int batchSize = operationBatch.getOperationCount();
+        metricJsonLog.addProperty(LogConstant.TRACE_ID, traceId);
+        metricJsonLog.addProperty(LogConstant.SUCCESS, succeed);
+        metricJsonLog.addProperty(LogConstant.BATCH_SIZE, batchSize);
+        metricJsonLog.addProperty(LogConstant.PARTITION_ID, partitionId);
+        metricJsonLog.addProperty(LogConstant.END_TIME, current);
```

Comment:
could build json be extracted to one utility method? I have seen it multiple times. I thought it would be better for maintainance.